docs/docs/reading-events/Serverside filtering.txt

```diff
@@ -0,0 +1,210 @@
+# Serverside filtering
+In the latest of EventStoreDB we brought out a new feature, server side filtering. This feature allows you to pass a filter when querying EventStoreDB so that it will only return events that you asked for. 
```

Comment:
```suggestion
The 20.06 release of EventStoreDB introduces a new feature: server side filtering. This feature allows you to pass a filter when querying EventStoreDB so that it will only return events that you asked for. 
```